### ISSUE TITLE
Bug:  Tab bar more than 5 crash the application

### ISSUE THREAD

**Issue by steve21124** [Jasonette/JASONETTE-iOS#75]
Tab bar more than 5 crash the application

error: Jasonette[33191:211960] *** Terminating app due to uncaught exception 'NSRangeException', reason: '*** -[__NSArrayI objectAtIndex:]: index 5 beyond bounds [0 .. 4]'
*** First throw call stack:
<PRE>
{
  "$jason": {
    "head": {
      "title": "Tabs Footer",
      "styles": {
        "bold": {
          "font": "HelveticaNeue-CondensedBold",
          "padding": "10",
          "size": "15"
        },
        "normal": {
          "font": "HelveticaNeue",
          "size": "12"
        }
      },
      "actions": {
        "$foreground": {
          "type": "$reload"
        },
        "send": {
          "type": "$util.alert",
          "options": {
            "title": "Message",
            "description": "{{$get.message}}"
          }
        }
      }
    },
    "body": {
      "header": {
        "menu": {
          "text": "View JSON",
          "href": {
            "url": "https://github.com/Jasonette/Jasonpedia/blob/gh-pages/view/footer/tabs.json",
            "view": "web"
          }
        }
      },
      "sections": [
        {
          "items": [
            {
              "type": "label",
              "text": "Fullscreen transition",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/action/audio/vaultboy/index.json",
                "transition": "fullscreen"
              }
            },
            {
              "type": "label",
              "text": "Next",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
              }
            },
            {
              "type": "label",
              "text": "Next",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
              }
            },
            {
              "type": "label",
              "text": "Next",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
              }
            },
            {
              "type": "label",
              "text": "Next",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
              }
            },
            {
              "type": "label",
              "text": "Next",
              "class": "bold",
              "href": {
                "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
              }
            }
          ]
        }
      ],
      "footer": {
        "tabs": {
          "style": {
            "background": "#ffffff"
          },
          "items": [
            {
              "image": "https://s3-us-west-2.amazonaws.com/www.jasonclient.org/topsecret%402x.png",
              "text": "Top Secret",
              "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
            },
            {
              "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/info%402x.png",
              "text": "Info",
              "badge": "2",
              "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
            },
            {
              "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/apps%402x.png",
              "text": "Collection",
              "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
            },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/www.jasonclient.org/topsecret%402x.png",
                    "text": "Top Secret",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/info%402x.png",
                    "text": "Info",
                    "badge": "2",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/apps%402x.png",
                    "text": "Collection",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/www.jasonclient.org/topsecret%402x.png",
                    "text": "Top Secret",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/info%402x.png",
                    "text": "Info",
                    "badge": "2",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    },
                    {
                    "image": "https://s3-us-west-2.amazonaws.com/fm.ethan.jason/apps%402x.png",
                    "text": "Collection",
                    "url": "https://jasonette.github.io/Jasonpedia/view/footer/tabs.json"
                    }                    
                    
          ]
        }
      }
    }
  }
}

</PRE>

**Comment by gliechtenstein** [comment]
Fixed by @wajahatch888  with https://github.com/Jasonette/JASONETTE-iOS/pull/85 🎉

Our decision was to NOT support more than 5 tab bar items at the moment, because supporting more than 5 has problems on the Android side as well, and trying to prematurely support this in a hackish manner will only add technical debt cross platform.

TLDR:  If the JSON contains more than 5 items, it will be cut off at 5. Only the first 5 tab bar items will show up.
